stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/internal/GiphyAttachmentViewHolder.kt

```diff
@@ -109,6 +110,22 @@ internal class GiphyAttachmentViewHolder(
 
         imageCorners(binding.mediaAttachmentView, data)
 
+        val width = with(binding.mediaAttachmentView) {
+            when {
+                !isStyleInitialized() -> ConstraintLayout.LayoutParams.MATCH_CONSTRAINT
```

Comment:
Not too happy with doing a "backwards read", however style should always be initialized here and if for some reason it goes awry the fallback is to just splay itself to the maximum width possible.